rotkehlchen/exchanges/coinbase.py

```diff
@@ -325,9 +428,6 @@ def _api_query(
                 gevent.sleep(.5)
                 continue  # do a single retry since they don't have info on retries
 
-            if response.status_code == 403:
```

Comment:
Are you absolutely certain that neither the legacy, nor the new API will never ever return a 403 HTTP API status? We had this there for a reason, and that is that theold API at some point did return 403.